docs/element-definitions.md

```diff
@@ -210,8 +219,8 @@ Note: At the moment only the HTTP protocol is supported.
 
         Definition of any input message-body attribute for this transition.
 
-    - `contentTypes` (array[String]) - A collection of content types that MAY be used for the transition.
-- `content` (array)
+    - `contentTypes` (Array Element[[String Element][]]) - A collection of content types that MAY be used for the transition.
+- `content` (array, fixed-type)
```

Comment:
Should this be an array element?

Otherwise I think this could be a problematic case when a client is parsing a `Transition` element when it does not know that the type is an array.